suites/apis/rasAuthN.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable codeceptjs/no-skipped-tests */
+/*
+ RAS Integration Test - AuthN Only (PXP-6505)
+ This test plan has a few pre-requisites:
+ 1. Fence > 4.23.0 must be deployed.
+ 2. There must be an upstream client previously registered with NIH/RAS and
+    the test environment must have its client & secret ID stored into its Fence config.
+    e.g., $ cat gen3_secrets_folder/configs/fence-config.yaml | yq .OPENID_CONNECT.ras
+          {
+              "client_id": "****",
+              "client_secret": "****",
+              "redirect_url": "{{BASE_URL}}/login/ras/callback"
```

Comment:
I guess we should update these comments to make clear that the `discovery_url` is now mandatory. 🤷